pkg/limayaml/validate.go

```diff
@@ -641,3 +655,36 @@ func ValidateAgainstLatestConfig(yNew, yLatest []byte) error {
 
 	return nil
 }
+
+func validateSocket(engine, socket string) error {
+	if socket == "" {
+		return fmt.Errorf("%s socket path must not be empty", engine)
+	}
```

Comment:
Use the standard format for validation errors:

```suggestion
func validateSocket(field, socket string) error {
	if socket == "" {
		return fmt.Errorf("field `%s` must not be empty", field)
	}
```